### PR TITLE
Ignition 8 troubleshooting/FAQ

### PR DESCRIPTION
**Pull request by TomKimsey**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -234,3 +235,21 @@ Any changes done on this page require a restart of the connector module or Ignit
...
+Your Talk2M data usage can be accessed through [eCatcher](https://ewon.biz/cloud-services/talk2m/ecatcher "https://ewon.biz/cloud-services/talk2m/ecatcher").
+
+1. Open eCatcher and login.
+2. On the left hand menu bar click `Account`.
+3. Once in the account properties click `Reports` near the top of the page.
+4. From here you will be able to download the current or past montly connection log reports.
+
+### If Ignition is not running will my data still be logged?
+
+If Ignition is offline data will still be logged by the Flexy and stored in Datamailbox. The amount of data points stored in Datamailbox depends on your [account type](https://ewon.biz/cloud-services/talk2m/plan-pricing "https://ewon.biz/cloud-services/talk2m/plan-pricing"). Once Ignition is online again all logged points will be fetched from Datamailbox by Ignition.
```
Comment (alexjhawk) [inline readme.md]: should have comma between "is offline" and "data will still..." as well as between "online again" and "logged points"

File: readme.md [hunk 2]
```diff
@@ -204,3 +205,32 @@ _Note: The realtime functionality counts against your Talk2M account's monthly b
...
+
+* [Tags do not appear in Ignition](#Tags-do-not-appear-in-Ignition)
+* [Changes on the Ewon Connector page do not take effect](#Changes-on-the-Ewon-Connector-page-do-not-take-effect)
+
+### Tags do not appear in Ignition
+
+There are several reasons tags may not be populated into Ignition. Please follow the following steps in order.
+
+1. Verify all steps in the [Ewon Setup](#ewon-setup) section were followed.
+2. If the system has just been configured wait one hour for data to be registered in Datamailbox.
```
Comment (alexjhawk) [inline readme.md]: comma between "configured" and "wait"

File: readme.md [hunk 3]
```diff
@@ -204,3 +205,32 @@ _Note: The realtime functionality counts against your Talk2M account's monthly b
...
+3. Verify your Talk2M credentials are correctly entered into the Ewon Connector settings in Ignition.
+   * If your credentials were incorrectly entered the Ignition logs will indicate a Authentication Error.  Check the logs on the Ignition webserver by clicking `Status` > `Logs`. If your credentials are incorrect you will see log entries in this format.
```
Comment (alexjhawk) [inline readme.md]: comma between "entered" and "the Ignition logs" and between "incorrect" and "you"

File: readme.md [hunk 4]
```diff
@@ -204,3 +205,32 @@ _Note: The realtime functionality counts against your Talk2M account's monthly b
...
+
+   <img src="https://raw.githubusercontent.com/hms-networks/IgnitionEwonConnector/Ignition-8-master/images/auth_error.JPG" alt="Ignition Authentication Error" width="1000"/>
+
+   * If you see this log error check your Ewon Account information entered on the `Ewon Connector` configuration page. _Note: Any changes done on this page require a restart of the connector module or Ignition. You can restart the module by naviation to `Config` > `Modules` and clicking `restart` next to the module._
```
Comment (alexjhawk) [inline readme.md]: comma between "log error" and "check your..."

File: readme.md [hunk 5]
```diff
@@ -204,3 +205,32 @@ _Note: The realtime functionality counts against your Talk2M account's monthly b
...
+4. If tag values are still not populated in Ignition please contact [support](#support)
```
Comment (alexjhawk) [inline readme.md]: command after "Ignition"

File: readme.md [hunk 6]
```diff
@@ -72,6 +73,41 @@ The module can be downloaded from the [releases page](https://github.com/hms-net
 
 <sup>[Back to top](#table-of-contents)</sup>
 
+## [Ewon Setup](#table-of-contents)
+
+When setting up an Ewon follow all standard Ewon Flexy documentation and Install guides. [https://websupport.ewon.biz/support](https://websupport.ewon.biz/support "https://websupport.ewon.biz/support")
```
Comment (alexjhawk) [inline readme.md]: comma after "Ewon"

File: readme.md [hunk 7]
```diff
@@ -72,6 +73,41 @@ The module can be downloaded from the [releases page](https://github.com/hms-net
...
+
+In addition to the standard documentation follow these additional steps.
```
Comment (alexjhawk) [inline readme.md]: comma after documentation

File: readme.md [hunk 8]
```diff
@@ -72,6 +73,41 @@ The module can be downloaded from the [releases page](https://github.com/hms-net
...
+
+>**Deadband Logging**
+>
+>The Logging Deadband field triggers logging when the tags value changes by a particular amount. For example, a logging deadband of 2 means that a tag will be logged when the value changes by 2 or more. A negative value in the logging deadband field disables logging on value change.
+>
+>**Interval Logging**
+>
+>The Logging Interval field triggers time based logging. For example, a logging interval of 60 means that the tag will be logged every 60 seconds. A logging interval of 0 disables time based logging.
+
+Once configured click `Update Tag` on the bottom right of the page.
```
Comment (alexjhawk) [inline readme.md]: comma after configured

File: readme.md [hunk 9]
```diff
@@ -72,6 +73,41 @@ The module can be downloaded from the [releases page](https://github.com/hms-net
...
+
+<img src="https://raw.githubusercontent.com/hms-networks/IgnitionEwonConnector/Ignition-8-master/images/tag_config_page.JPG" alt="Ewon Tag Configuration Page" width="1000"/>
+
+### Enable Historical Data In DataMailbox
+
+Open the Ewon webserver and navigate to the `Data Management` page ( On the left hand menu click `Setup` > `System` > `Main` > `Data Management`). Check the `Historical Data` checkbox and set the `Datamailbox upload interval` to a reasonable number for your application. The `Datamailbox upload interval` sets how often the Flexy will send batches of historical data to Datamailbox and does not affect the frequency at which tag values are logged. When done click `Update` at the bottom of the page.
```
Comment (alexjhawk) [inline readme.md]: comma between "When done" and "click"

**Comment by TomKimsey** [comment]
Added a million commas.

**Pull request merged**